src/txkube/_memory.py

```diff
@@ -104,6 +105,15 @@ def response(request, status, obj):
 
 
 
+def _message(details, event):
+    if details.get(u"group") is None:
+        fmt = u"{kind} \"{name}\" {event}"
```

Comment:
Yea that makes sense ... double quoting everything is mostly just a habit.  Cases where single quotes actually improve readability don't come up very often, didn't occur to me here. :)